components/widgets/inputs/users_emails_input.jsx

```diff
@@ -197,13 +258,14 @@ export default class UsersEmailsInput extends React.Component {
         });
         return (
             <AsyncSelect
+                ref={this.selectRef}
                 styles={this.customStyles}
                 onChange={this.onChange}
-                loadOptions={this.props.usersLoader}
-                isValidNewOption={isEmail}
+                loadOptions={this.optionsLoader}
+                isValidNewOption={this.showAddEmail}
                 isMulti={true}
                 isClearable={false}
-                className='UsersEmailsInput'
+                className={'UsersEmailsInput ' + (this.props.inputValue === '' ? 'empty' : '')}
```

Comment:
Could we use the `classnames` import to simplify this logic, and make the resulting classname more explicit (and greppable!)

Reply:
I didn't know classnames, but yes, sure we can :)

Reply:
Done